src/disc/stk/Albany_GenericSTKMeshStruct.cpp

```diff
@@ -443,91 +376,6 @@ void GenericSTKMeshStruct::setDefaultCoordinates3d ()
   }
 }
 
-void GenericSTKMeshStruct::rebalanceInitialMeshT(const Teuchos::RCP<const Teuchos::Comm<int> >& comm){
```

Comment:
Are we completely getting rid of the rebalance capability? We might need it for GMesh

Reply:
No problem - I'll put this back if we think we might need it. It might be better to just replace this with the stk_balance capability (in the Trilinos stk package) as we are using some pretty old stuff down in the percept directory of Albany for this.

Please say the word and I'll put this back the way it was.

Reply:
Not sure.. @bartgol what do you think? You are the only one using rebalance for GMesh.

Reply:
I didn't spend a lot of time looking at this, but there is an integration test for the rebalance in:

stk_integration_tests/stk_balance/IntegrationTestLoadBalance.cpp

Looks pretty interesting at a glance.

Reply:
I think someone else was using GMesh, I think @JustinClough maybe? Anyhow, I would like to keep GMesh, since it's a very user-friendly meshing tool to create non cartesian geometries (even a simple circle).

If we really want to get rid of the rebalance feature, then we should probably create a mini-executable, to create exodus meshes from GMesh inputs (to be run in serial mode). It shouldn't be complicated, but might take some time. If it's not too much effort, maybe we can just maintain rebalance for the time being?

Reply:
I put this back in the latest push.